README.md

```diff
@@ -37,14 +38,15 @@ import (
 func main() {
     database, _ := sql.Open("mysql", "/")
 	mysql := db.NewMySQLChecker(database)
+    timeout := 5 * time.Second
     
     companies := health.NewCompositeChecker()
-    companies.AddChecker("Microsoft", url.NewChecker("https://www.microsoft.com/"))
-    companies.AddChecker("Oracle", url.NewChecker("https://www.oracle.com/"))
-    companies.AddChecker("Google", url.NewChecker("https://www.google.com/"))
+    companies.AddChecker("Microsoft", url.NewChecker("https://www.microsoft.com/", timeout))
```

Comment:
I think we should not break the API. I prefer keep the url.NewChecker intact and create another url.NewCheckerWithTimeout and add this second parameter.